src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -435,3 +437,20 @@ def _get_local_storage(self, run: Union[str, Run]) -> LocalStorageOperations:
         if isinstance(run, str):
             run = self.get(name=run)
         return LocalStorageOperations(run)
+
+    def _get_create_or_update_telemetry_values(self, run: Run, **kwargs):
+        flow_obj = load_flow(source=run.flow)
+        if isinstance(flow_obj, EagerFlow):
+            return {"flow_type": "eager"}
```

Comment:
Give these two type some constants. 
Let's call them "DAG" and "FLEX"

Reply:
ok, I add the enum and the value is "DAG", "FLEX", but i still keep the keys are "YAML_FLOW", "EAGER_FLOW" Because of they are concepts that are easier to understand and consistent with other parts of the code.